score-lang-api/src/main/java/org/openscore/lang/api/Slang.java

```diff
@@ -26,25 +26,16 @@
 public interface Slang {
 
     /**
-     * Compile a flow written in slang
+     * Compile a flow/operation written in slang
      * @param source the slang source containing the flow
-     * @param dependencies a set of slang sources of of all the flow's dependencies
+     * @param dependencies a set of slang sources of of all the flow/operation's dependencies
      * @return the compiled artifact of the flow
      */
     public CompilationArtifact compile(SlangSource source, Set<SlangSource> dependencies);
 
-    /**
-     * Compile an operation written in slang
-     * @param source the slang source containing the operation
-     * @param operationName the name of the operation to compile from the source
-     * @param dependencies a set of slang sources of all the operation's dependencies
-     * @return the compiled artifact of the operation
-     */
-    public CompilationArtifact compileOperation(SlangSource source, String operationName, Set<SlangSource> dependencies);
-
 	/**
 	 * Run a flow or operation written in slang already compiled to a compilationArtifact
-	 * @param compilationArtifact the compiled artifact of the flow
+	 * @param compilationArtifact the compiled artifact of the flow/operation
```

Comment:
as @SamMarkowitz commented on another API it's preferable to use 'or' and not a slash